docs/topics/kmm/kmm-getting-started.md

```diff
@@ -1,22 +1,55 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: Simplify cross-platform app development with Kotlin Multiplatform Mobile. Create a single codebase
+for the business logic of your iOS and Android apps.)
 
-## Get familiar with KMM
+> Kotlin Multiplatform Mobile is in [Alpha](components-stability.md). Language features and tooling may change in future
+> Kotlin versions.
+> 
+> Beta is expected in spring 2022. See [KMM Beta Roadmap](https://youtu.be/LE-Dy9XCLxk) to learn about upcoming features.
+>
+{type="note"}
 
-Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Developer Advocate Ekaterina 
-Petrova will explain what Kotlin Multiplatform Mobile is and how you can use it in your project. Together with Ekaterina 
-you'll set up an environment and prepare for creating your first cross-platform mobile application with KMM.
+Kotlin Multiplatform Mobile (KMM) is an SDK designed to simplify the development of cross-platform mobile applications.
+You can share common code between iOS and Android apps and write platform-specific code only where it's necessary.
+For example, to implement a native UI or when working with platform-specific APIs.
+
+Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Product Marketing Manager Ekaterina 
+Petrova explains what Kotlin Multiplatform Mobile is and how to use it in your project. With Ekaterina,
+you'll set up an environment and prepare to create your first cross-platform mobile application:
 
 <video href="mdN6P6RI__k" title="Kotlin Multiplatform Multiverse, Episode 1: Meet KMM!"/>
 
-## Start KMM from scratch
+You can also check out other videos about [Kotlin Multiplatform Multiverse](https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C) on YouTube.
+
+## Supported platforms
+
+* Android applications and libraries
+* [Android NDK](https://developer.android.com/ndk) (ARM64 and ARM32)
+* Apple iOS devices (ARM64 and ARM32) and simulators
+* Apple watchOS devices (ARM64 and ARM32) and simulators
+
+[Kotlin Multiplatform](multiplatform.md) technology also supports other platforms, including JavaScript, Linux, WebAssembly,
+and [more](mpp-dsl-reference.md#targets).
+
+See the [full list of supported platforms](mpp-supported-platforms.md) for Kotlin Multiplatform programming.
```

Comment:
I would combine the last two paragraphs or maybe even include the link to platform descriptions in the first paragraph.
It'd help keep shorter this section with additional information.

Reply:
True, targets in MPP DSL reference and Supported platforms seem to have similar information. Left one link only and combined into one paragraph